app/src/main/java/ch/epfl/sdp/peakar/user/services/RemoteAuthAccount.java

```diff
@@ -0,0 +1,210 @@
+package ch.epfl.sdp.peakar.user.services;
+
+import android.util.Log;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import ch.epfl.sdp.peakar.database.Database;
+import ch.epfl.sdp.peakar.database.DatabaseReference;
+import ch.epfl.sdp.peakar.database.DatabaseSnapshot;
+import ch.epfl.sdp.peakar.general.remote.RemoteOutcome;
+import ch.epfl.sdp.peakar.general.remote.RemoteResource;
+import ch.epfl.sdp.peakar.points.CountryHighPoint;
+import ch.epfl.sdp.peakar.points.POIPoint;
+import ch.epfl.sdp.peakar.user.outcome.ProfileOutcome;
+import ch.epfl.sdp.peakar.user.friends.RemoteFriendItem;
+
+/**
+ * This class extends AuthAccount to handle operations on a Database
+ */
+public class RemoteAuthAccount extends AuthAccount implements RemoteResource {
+    private static String currentID = null;
+    private static RemoteAuthAccount instance = null;
+    private static DatabaseReference dbRefUser = null;
+
+    /**
+     * Create a new account instance.
+     * @param newID ID of the authenticated user
+     */
+    private RemoteAuthAccount(String newID) {
+        dbRefUser = Database.getInstance().getReference().child(Database.CHILD_USERS + newID);
+        currentID = newID;
+    }
+
+    /**
+     * Get a Firebase account instance.
+     * WARNING: you SHOULD NOT call this method. To get an account you should call <code>Account.getInstance(String authID)</code>.
+     * @param authID id of the user.
+     */
```

Comment:
`@return` is missing

Reply:
The first line of the doc already says what is returned, should I write it once more anyway?